app/scripts/common/components/oo-email-dialog.html

```diff
@@ -38,7 +38,7 @@ <h3 class="panel-title">
             <label for="attachment" translate>Rechnung anhängen</label>
         </div>
         <div class="col-md-6">
-            <input bs-switch name="attachInvoice" ng-model="selectedMailTemplate.attachInvoice" type="checkbox" switch-size="medium" switch-animate="true" switch-on-text="{{'Ja'|translate}}" switch-off-text="{{'Nein'|translate}}" switch-on-color="success" switch-off-color="danger" switch-radio-off="false" ng-true-value="true" ng-false-value="false">
+            <input bs-switch name="attachInvoice" ng-model="selectedMailTemplate.attachInvoice" type="checkbox" switch-size="medium" switch-animate="true" switch-on-text="{{'Ja'|translate}}" switch-off-text="{{'Nein'|translate}}" switch-on-color="success" switch-off-color="danger" switch-radio-off="false" ng-true-value="true" ng-false-value="false" checked>
```

Comment:
Does this change the model-value right on rendering?

Reply:
yes, this is the one that force the true. I have also changed the model to true to keep it consistent